release-notes-3.3.0.md

```diff
@@ -4,9 +4,8 @@
 ${version-number}
 
 #### New Features
-- SCMOD-11949: The [gosu](https://github.com/tianon/gosu/) tool is now available in the base image.  
-  This tool allows derived images to run commands as a specified user, rather than as the default user.  
-  See the [README.md](https://github.com/CAFapi/opensuse-elasticsearch7-image/blob/master/README.md) for more details.
+- SCMOD-12755: The pre-installed startup scripts no longer requires root permissions to run. This allows containers
+  built on this image to run as arbitrary users.
```

Comment:
Is this true for this image?

Reply:
Not at the moment, at least until https://portal.digitalsafe.net/browse/SCMOD-12882 is done

Reply:
I'm confused that you're saying this is not true but haven't removed it.